packages/test/test-service-load/src/optionsMatrix.ts

```diff
@@ -58,7 +58,7 @@ const gcOptionsMatrix: OptionsMatrix<IGCRuntimeOptions> = {
 };
 
 const summaryOptionsMatrix: OptionsMatrix<ISummaryRuntimeOptions> = {
-    disableIsolatedChannels: booleanCases,
+    disableIsolatedChannels: [false],
```

Comment:
Maybe just remove it? So it takes whatever is the default.

Reply:
you cannot remove it. the options matrix requires all fields be set. this is so new options cause a break and must have their values explicitly set.

Reply:
Ah okay. I wasn't aware of this :-).

Reply:
you should be able to set the value to undefined, which is essentially default in this case